frontend/src/app/commonComponents/TextWithTooltip.tsx

```diff
@@ -49,7 +49,11 @@ export const TextWithTooltip = ({
       wrapperclasses="usa-text-with-tooltip"
     >
       {text}
-      <FontAwesomeIcon className="info-circle-icon" icon={faInfoCircle} />
+      <FontAwesomeIcon
+        alt-text="info"
```

Comment:
I would've thought that the alt text here should be "info icon", but in checking Deque it seems that is not their recommendation: https://dequeuniversity.com/class/images/icon-font/informative-icon-fonts

Just sharing in case anyone else was curious!

(Screenshot for those without Deque access)
![Screen Shot 2022-10-04 at 4 09 41 PM](https://user-images.githubusercontent.com/80282552/193947092-f26dba1e-7102-4f57-b624-853c0d4639e9.png)

Reply:
@emmastephenson This is interesting! It was also my understanding that the `alt-text` should be descriptive rather than literal, so this is a good confirmation.

Related to this example, it seems to also confirm there are two ways to work with interactive icons, which is either adding alt-text or using aria-label. I opted to use alt-text because the default aria attributes for FontAwesomeIcons were `aria-hidden="true"` and `focusable="false"` and I wanted to avoid overriding a bunch of defaults. Again, I think this is ok in our case because we wrap the icons in a button, unlike this example. A little confusing though. Does that sound right to you?

Reply:
Yep, I think you're exactly right. With the FontAwesome icon setting the icon to `aria-hidden="true"`, `aria-label` wouldn't be read out for screenreader users. Because it's classified as an `img`, I think `alt-text` works well.